### ISSUE TITLE
anki-forvo-dl not working after update to Anki 23.10

### ISSUE THREAD

**Issue by hu3rror** [realmayus/anki_forvo_dl#97]
anki-forvo-dl stopped working after I updated to the latest Anki version 23.10. It shows the following error:

```
Anki 23.10 (51a10f09) Python 3.9.15 Qt 6.5.3 PyQt 6.5.3
Platform: Windows-10-10.0.22621
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-11-01 09:56:04

When loading anki-forvo-dl - Forvo Pronunciation Downloader fully automatic:
Traceback (most recent call last):
  File "aqt.addons", line 244, in loadAddons
  File "C:\Users\user\Apps\Scoop\persist\anki\data\addons21\858591644\__init__.py", line 14, in <module>
    from .src.About import About
  File "C:\Users\user\Apps\Scoop\persist\anki\data\addons21\858591644\src\About.py", line 1, in <module>
    from PyQt5.QtWidgets import QDialog, QLabel, QVBoxLayout
ModuleNotFoundError: No module named 'PyQt5'
```

Could the add-on be updated to support the latest Anki version? Let me know if any other details would be helpful!

**Comment by orangeqoon** [comment]
Doesn't work either yet. pls updata

**Comment by realmayus** [comment]
Fixed with #101
